Source/Scene/Cesium3DTileFeatureTableResources.js

```diff
@@ -0,0 +1,82 @@
+/*global define*/
+define([
+        '../Core/ComponentDataType',
```

Comment:
Should be `../Core/ComponentDatatype` and below. This was causing the karma build to fail.